packages/contracts-bedrock/scripts/DeploySuperchain.s.sol

```diff
@@ -117,17 +117,17 @@ contract DeploySuperchainInput is BaseDeployIO {
         string memory toml = vm.readFile(_infile);
 
         // Parse and set role inputs.
-        set(this.guardian.selector, toml.readAddress(".roles.guardian"));
-        set(this.protocolVersionsOwner.selector, toml.readAddress(".roles.protocolVersionsOwner"));
-        set(this.proxyAdminOwner.selector, toml.readAddress(".roles.proxyAdminOwner"));
+        set(this.guardian.selector, toml.readAddress(".dsi.roles.guardian"));
+        set(this.protocolVersionsOwner.selector, toml.readAddress(".dsi.roles.protocolVersionsOwner"));
+        set(this.proxyAdminOwner.selector, toml.readAddress(".dsi.roles.proxyAdminOwner"));
 
         // Parse and set other inputs.
-        set(this.paused.selector, toml.readBool(".paused"));
+        set(this.paused.selector, toml.readBool(".dsi.paused"));
```

Comment:
We should have `paused` default to false if not present. I'll defer to you on whether you want to add defaults in this PR or do them all in a future PR.

You can do this using the `vm.keyExistsToml` cheat, something like this:
```solidity
bool paused = vm.keyExistsToml(toml, ".paused") ? toml.readBool(".dsi.paused") : false;
set(this.paused.selector, paused);
```

Reply:
Okay, I added this suggestion in this PR: https://github.com/ethereum-optimism/optimism/pull/11975/commits/d11660e77476e00456829dc3a37fb499d65328e9#diff-62b3d04f126e442fa8b92be128f8e5a03d3470e09d65f5f5657c5d0f842eff90R127 

But I've raised this [issue](https://github.com/ethereum-optimism/optimism/issues/12008) (and tracked in figjam) to add more of these. Will need to know what all the default values should be before I take on that issue. I'm thinking the standard config might come in useful here: https://specs.optimism.io/protocol/configurability.html